oraclizeAPI_0.4.25.sol

```diff
@@ -1029,7 +5120,13 @@ contract usingOraclize {
         }
     }
 
-    function oraclize_randomDS_proofVerify__sessionKeyValidity(bytes proof, uint sig2offset) internal returns (bool) {
+    function oraclize_randomDS_proofVerify__sessionKeyValidity(
+        bytes proof,
+        uint sig2offset
```

Comment:
Don't know if it is worth to mention or correct every `uint` to `uint256`

Reply:
Ignoring all `uints` in `0.4.25. Those in `0.5` are done. Ignoring remaining mentions.